soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -46,15 +48,12 @@ namespace GameControlEditor {
         }
     }
 
-    void DrawHelpIcon(const std::string& helptext, bool sameline = true, int Pos = 0) {
+    void DrawHelpIcon(const std::string& helptext) {
         // place the ? button to the most of the right side of the cell it is using.
+        ImGui::SetCursorPosY(ImGui::GetCursorPosY() - 22);
         ImGui::SetCursorPosX(ImGui::GetCursorPosX() + ImGui::GetContentRegionAvail().x - 15);
         ImGui::SmallButton("?");
         SohImGui::Tooltip(helptext.c_str());
-        if (sameline) {
-            //I do not use ImGui::SameLine(); because it make some element vanish.
-            ImGui::SetCursorPosY(ImGui::GetCursorPosY() - 22);
-        }
     }
```

Comment:
this also feels unrelated to the walk speed modifiers, not sure what this changes

Reply:
It's part of the fix necessary for the tooltips to actually work in this custom control window; they currently don't after #1212.  I didn't want to make a separate PR for this because I would conflict with myself due to the other work I did in the custom control window for this PR.

Reply:
coolcool, might be nice to add a couple bullet points to the main PR comment with a high level description of the changes and/or make an issue for the custom control window tooltip stuff and link it

Reply:
Good call, I haven't touched that PR description in a few... months, so I'll add that info there.